app/javascript/react/components/NewMovieFormTile.js

```diff
@@ -0,0 +1,82 @@
+import React, {useState} from "react"
+import ErrorList from "./ErrorList";
+import _ from "lodash"
+
+const NewMovieFormTile = (props) => {
+
+  const [errors, setErrors] = useState({})
+  const [movieRecord, setMovieRecord] = useState({
+    title: "",
+    year: "",
+    director: "",
+    image: "",
+    description: ""
+  })
+
+  const submitHandler = async (event) => {
+    event.preventDefault()
+    if (validForSubmission()) {
+      props.addMovie(movieRecord)
+    }
+  }
+
+  const handleInputChange = (event) => {
+    setMovieRecord({
+      ...movieRecord,
+      [event.currentTarget.name]: event.currentTarget.value
+    })
+  }
+
+  const validForSubmission = () => {
+    let submitErrors = {}
+    const requiredFields = ["title", "year"]
+    requiredFields.forEach(field => {
+      if (movieRecord[field].trim() === "") {
+        submitErrors = {
+          ...submitErrors,
+          [field]: "is blank"
+        }
+      }
+    })
+  
+    setErrors(submitErrors)
+    return _.isEmpty(submitErrors)
+  }
+
+  return (   
+    <form className="callout" onSubmit={submitHandler}>
+      <ErrorList errors={errors}/>
+
+      <label htmlFor="title">
+        Title
+        <input type="text" name="title" onChange={handleInputChange} value={movieRecord.title}/>
+      </label>
+
+      <label htmlFor="year">
+        Year
+        <input type="text" name="year" onChange={handleInputChange} value={movieRecord.year}/>
+      </label>
+
+      <label htmlFor="director">
+        Director
+        <input type="text" name="director" onChange={handleInputChange} value={movieRecord.director}/>
+      </label>
+
+      <label htmlFor="image">
+        Image
+        <input type="text" name="image" onChange={handleInputChange} value={movieRecord.image}/>
```

Comment:
doesn't look like file uploading feature has been implemented yet - this input field for an image should be removed until ready to work on image uploading so that a drop zone can be added and that the AWS S3 bucket has been set up to store the images. Image uploading is intended to be its own separate feature from the initial form. It's best to first make sure everything else in the form is working first, then when ready to implement image uploading we will come back to update the form.